### PR TITLE
AG-12428 CSS docs

### PR DESCRIPTION
**Pull request by jacobp100**
https://ag-grid.atlassian.net/browse/AG-12428

### CODE REVIEW

File: packages/ag-charts-website/src/content/docs/layout/_examples/chart-class/styles.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.chart {
```
Comment (markdurrant) [inline packages/ag-charts-website/src/content/docs/layout/_examples/chart-class/styles.css:1]: I don't think this class is actually referenced by the example. When I run the example locally I don't see this file in the files list, and no red border.

**Review by markdurrant — CHANGES_REQUESTED** [review]
Generally looking good, I think the page is much clearer without the tabs. 

Only issue is the CSS class example not picking up the new `styles.css` file.

File: packages/ag-charts-website/src/content/docs/layout/_examples/chart-class/styles.css (hunk 1 above)
Comment (jacobp100) [reply packages/ag-charts-website/src/content/docs/layout/_examples/chart-class/styles.css:1]: I think you have to save this file to force it to rebuild the example

Comment (jacobp100) [reply packages/ag-charts-website/src/content/docs/layout/_examples/chart-class/styles.css:1]: Definitely worked locally for me - maybe it's easier to just revisit if it doesn't show up on staging

Comment (markdurrant) [reply packages/ag-charts-website/src/content/docs/layout/_examples/chart-class/styles.css:1]: Sorry my bad. Wasn't aware about the save thing. Can confirm all works locally for me. Everything looking great 👍

**Review by markdurrant — APPROVED** [review]
LGTM 👍

**Review by alantreadway — APPROVED** [review]

**Pull request merged**